### DISCUSSION TITLE
[checklist] Separate status columns for each platform

### DISCUSSION THREAD

**Discussion by fujiokayu** [OWASP/mastg#2098] [Ideas]
The current (ver 1.4.1) checklist has only one status column. 
In my humble opinion, it would be easier to use if we have separate columns for Android and iOS, since we would need two checksheets for each assessment.

- current: 
<img width="393" alt="image" src="https://user-images.githubusercontent.com/17287988/160956923-12450a07-75f7-43dd-b118-1964004f643e.png">

- my opinion: 
<img width="419" alt="image" src="https://user-images.githubusercontent.com/17287988/160958180-78512d57-618c-48f1-a1ec-519772d28384.png">

This can be fixed with a small modification to tools/scripts/yaml_to_excel.py.

**Comment by cpholguera** [comment]
Thanks for the feedback @fujiokayu, we'll think about it 😊

**Comment by cpholguera** [comment]
Covered in #2336

**Comment by fujiokayu** [comment]
@cpholguera 

#2341 has been merged, so it is OK to close this discussion.

**Comment by cpholguera** [comment]
Thank you @fujiokayu!
